### PR TITLE
Migrate from Azure Function to AWS

### PR DESCRIPTION
**Pull request by nojaf**
Pure out of interest, I'd like to migrate the existing Azure Functions to AWS Lambda.
@RobVanPamel, @RobKenis, @LodeKennes feel free to review.
The most relevant change for you will probably `infrastructure/Program.fs`, where all the AWS magic happens.

### CODE REVIEW

File: infrastructure/Program.fs [hunk 1]
```diff
@@ -48,122 +52,202 @@ let private getLastCommit () =
...
+                )
+
+            Iam.RolePolicy("fantomas-log-policy", args)
+
+        let gateway =
+            let cors =
+                ApiGatewayV2.Inputs.ApiCorsConfigurationArgs(
+                    AllowHeaders = inputList [ input "*" ],
+                    AllowMethods = inputList [ input "*" ],
+                    AllowOrigins = inputList [ input "*" ]
```
Comment (robvanpamel) [inline infrastructure/Program.fs:171]: Do you want to allow all origins are only from https://fsprojects.github.io ?

File: src/server/FantomasOnlinePreview/FantomasOnlinePreview.fsproj [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
   </PropertyGroup>
-  <ItemGroup>
-    <Content Include="host.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
   <ItemGroup>
     <None Include="paket.references" />
     <Compile Include="../../shared/FantomasOnlineShared.fs" />
+    <Compile Include="..\HttpConstants.fs" />
```
Comment (robvanpamel) [inline src/server/FantomasOnlinePreview/FantomasOnlinePreview.fsproj:14]: @nojaf I saw in multiple files that you are mixing backslashes with forward slashes ... I assume it just works fine but is seems a bit dirty, no?

**Review by robvanpamel — COMMENTED** [review]
It is looking good! Going for AWS 😄

File: infrastructure/Program.fs (hunk 1 above)
Comment (nojaf) [reply infrastructure/Program.fs:171]: Yeah, I guess I could thighten that a bit indeed.

File: src/server/FantomasOnlinePreview/FantomasOnlinePreview.fsproj (hunk 2 above)
Comment (nojaf) [reply src/server/FantomasOnlinePreview/FantomasOnlinePreview.fsproj:14]: Yeah you have point here, but for this project I'll let it slide 🙈

**Pull request merged**